data/metainfo/org.gnome.Software.metainfo.xml.in

```diff
@@ -66,1961 +66,7 @@
    Validate with `appstreamcli validate *.metainfo.xml`
   -->
   <releases>
-    <release date="2022-09-16" version="43.0" type="stable">
-      <description>
```

Comment:
I dug into this one a bit more. Since I last investigated a few things have changed. I have learned:

- Not extracting release notes for translation on damned lies is [a deliberate decision](https://gitlab.gnome.org/Infrastructure/damned-lies/-/issues/149) ([implemented here](https://gitlab.gnome.org/Infrastructure/damned-lies/-/commit/b2c54ed2b64060a1f4c8adbd410bb8e4d86fcd54)) rather than an oversight. I disagree in principle… but at the same time my original patch to remove all the release notes was also motivated by not wanting to pay the cost of translating thousands of words (financially in our case rather than in the time of volunteers) so it's hard to argue here! 
- Most of the `<description>`s in this file were marked as not translatable. See https://gitlab.gnome.org/GNOME/gnome-software/-/issues/1340 and 972404d65d7926293a34ce93af256345a3f968d5

So at the very least I think we should update the commit message here to give an accurate explanation of why there are no upstream translations of the release notes.

We could also consider adding the ` translatable="no"` attribute on each remaining `<description>` rather than deleting the notes entirely.

Reply:
I think removing the entire release note is easier from a rebase perspective, because it's a single rebase conflict compared to potentially multiple smaller conflicts. (This doesn't matter, after all, because these notes are not accessible.)

In any case, updated the commit message.